paddle/operators/batch_norm_op.cc

```diff
@@ -0,0 +1,424 @@
+/* Copyright (c) 2016 PaddlePaddle Authors. All Rights Reserve.
```

Comment:
Reserve => Reserved.

Reply:
all the code is wrong now, I will update them in another PR